float32 velocidadViento
float32 temperatura
float32 humedad
